buttonhub.py

```diff
@@ -83,6 +93,14 @@ def handle_request(device_id):
     return ''
 
 
+@app.route('/status')
+def get_status():
+    return {
+        'state': app_state,
```

Comment:
app_state is the stored state of the entire mqtt network and can be quite big. If you're just here for the battery levels it might be better to move this into a separate endpoint. Besides, if an app needed this, it might make sense to have it listen to mqtt directly.

We could also consider publishing the device status via mqtt.